operator-common/src/main/java/io/strimzi/operator/common/operator/resource/ConfigMapOperator.java

```diff
@@ -34,8 +34,16 @@ public ConfigMapOperator(Vertx vertx, KubernetesClient client) {
         return client.configMaps();
     }
 
+    public static final String ANCILLARY_CM_KEY_METRICS = "metrics-config.yml";
+
     @Override
     protected Future<ReconcileResult<ConfigMap>> internalPatch(String namespace, String name, ConfigMap current, ConfigMap desired) {
+        if (current.getData().get(ANCILLARY_CM_KEY_METRICS) != null && desired.getData().get(ANCILLARY_CM_KEY_METRICS) != null &&
```

Comment:
I do not think this would work. The problem is, that at the same time you need to make sure that the other fields are not changed. I can do a single change which would modify both metrics and logging. And in that case this would skip the rolling update needed for logs. I wonder if we need to use the `JsonDiff` in the same way as we do it for Satteful Sets (in `StatefulSetDiff`).